VATSpy.dat

```diff
@@ -2666,7 +2665,7 @@ ESMI|Sjobo Sovde|55.598333|13.677222||ESMM|0
 ESMJ|Kagerod|55.995|13.053333||ESMM|0
 ESMK|Kristianstad|55.920556|14.085278||ESMM|0
 ESML|Landskrona|55.944444|12.869444||ESMM|0
-ESMO|Oskarshamn|57.351944|16.498333||ESOS|0
+ESMO|Oskarshamn|57.351944|16.498333||ESMM|0

```

Comment:
Why has this been changed to ESMM? The airport belongs to ESOS.

Reply:
Sorry, got the wrong line, it's ESMP to be changed